apps/bfd-db-migrator/src/main/assembly/bin/bfd-db-migrator.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
+set -x
 scriptDirectory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 bfdDBMigratorAppJar="$(echo ${scriptDirectory}/bfd-db-migrator*.jar)"
 classpath="${bfdDBMigratorAppJar}:${scriptDirectory}/lib/*"
 mainClass="gov.cms.bfd.migrator.app.MigratorApp"
 [ -n "$JAVA_HOME" ] && "javaExecutable=${JAVA_HOME}/bin/java" || javaExecutable=java
 
 exec "$javaExecutable" -cp "$classpath" "$@" "$mainClass"
+set +x
```

Comment:
Probably just some debugging lines left in place...
```suggestion
scriptDirectory="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
bfdDBMigratorAppJar="$(echo ${scriptDirectory}/bfd-db-migrator*.jar)"
classpath="${bfdDBMigratorAppJar}:${scriptDirectory}/lib/*"
mainClass="gov.cms.bfd.migrator.app.MigratorApp"
[ -n "$JAVA_HOME" ] && "javaExecutable=${JAVA_HOME}/bin/java" || javaExecutable=java

exec "$javaExecutable" -cp "$classpath" "$@" "$mainClass"
```